File: ApplicationBaseDALInterface/Base/ICalendarData.vb
Public Interface ICalendarData
	Function GetCalendarData(ByVal BUSINESS_UNIT_SEQ_ID As Integer, ByVal Calendar_Name As String, ByRef dsCalendarData As DataSet) As Boolean
	Function SaveCalendarData(ByVal BUSINESS_UNIT_SEQ_ID As Integer, ByVal Calendar_Name As String, ByVal Entry As String, ByVal EntryDate As Date, Optional ByVal Account_Seq_id As Integer = 1) As Boolean
	Function DeleteCalendarData(ByVal BUSINESS_UNIT_SEQ_ID As Integer, ByVal Calendar_Name As String, ByVal Entry As String, ByVal EntryDate As Date, Optional ByVal Account_Seq_id As Integer = 1) As Boolean
End Interface

File: Core/DataAccess/Interfaces/ICalendarData.cs
using GrowthWare.DataAccess.Interfaces.Base;
using System;
using System.Data;

namespace GrowthWare.DataAccess.Interfaces
{
    public interface ICalendarData : IDBInteraction
    {
        string CalendarName {get; set;}
        int SecurityEntitySeqID {get; set;}

		bool GetCalendarData(ref DataSet calendarDataSet);
		bool SaveCalendarData(String comment, DateTime entryDate, int accountSeqId);
		bool DeleteCalendarData(String comment, DateTime entryDate, int accountSeqId);
    }
}
